dev-tools/README.md

```diff
@@ -1,5 +1,8 @@
 # Development Tools
 
+*NB* The notes and scripts in this directory are obsolete since the migration
```

Comment:
Do we want to handle this now? Or is there a task created for that?

Reply:
I'd rather not handle this now. I will create a task. (As far as I know only one person uses these scripts and they're not tested on the CI.)